README.md

```diff
@@ -68,11 +68,61 @@ When using **Github deploy keys**, GitHub servers will accept the _first_ known
 
 To support picking the right key in this use case, this action scans _key comments_ and will set up extra Git and SSH configuration to make things work.
 
-1. When creating the deploy key for a repository like `git@github.com:owner/repo.git` or `https://github.com/owner/repo`, put that URL into the key comment.
+1. When creating the deploy key for a repository like `git@github.com:owner/repo.git` or `https://github.com/owner/repo`, put that URL into the key comment. You can do so using this command: `ssh-keygen -b 4096 -t rsa -N "" -f key -C "git@github.com:my-org/my-swift-package-repo.git"`
```

Comment:
It's good to have this hint here :+1:.

At https://github.com/webfactory/ssh-agent#creating-ssh-keys, we show several variants of creating the key. So, maybe we should just have something like 

> You can do so by adding something like `ssh-keygen ... -C "git@github.com:my-org/my-repo.git"` to your `ssh-keygen` command.

Note this section here should have nothing to do with Swift.